server/jetty-app/README.md

```diff
@@ -29,19 +26,90 @@ To create a more production-like environment, you can create and invoke the star
 ./server/jetty-app/build/install/server-jetty/bin/start
 ```
 
+The specifics of the start script can be found in [unixStartScript.txt](../../buildSrc/src/main/resources/unixStartScript.txt),
+which has been adapted from the [gradle java application plugin](https://docs.gradle.org/current/userguide/application_plugin.html).
+
 ### Configuration
 
-The `START_OPTS` environment variable is used to set JVM arguments. For example:
+#### Quick
+
+The following is a list of some "quick" gradle properties that can be set to change the behavior of the Deephaven server:
+
+* `groovy`: use a groovy console (as opposed to python)
+* `anonymous`: use anonymous authentication
+* `debug`: enable JVM debug port 5005
+* `devCerts`: use development certificates and port 8443
```

Comment:
devMTLS too? Not sure where the line should be for important ones to mention

Reply:
Added `devMTLS` and `quiet`. I think the other ones (mostly excludes) are okay to leave out?